### ISSUE TITLE
Need company names or custom word list.

### ISSUE THREAD

**Issue by ggb667** [mdeanda/lorem#17]
This is pretty good, but it would be nice if the loreum.txt file could be replaced by feeding the Constructor a file.  I wanted to have company name type things in there like arrow star micro computer soft com etc.  or maybe if we could make custom categories and feed them words.

**Comment by mdeanda** [comment]
That sounds useful. Let me add that but I'll have to remember how to publish the update to maven as its been awhile.

**Comment by mdeanda** [comment]
Yes, feel free to implement. I'd suggest not using real names but perhaps a
list of adjective and another if nouns then select adj+noun.

Or perhaps made up "noun sounding words".

**Comment by GauthamArunachalam** [comment]
Hi @mdeanda I have implemented the custom category option. Let me know if you have any review comments. 

@ggb667 if you are free kindly have a look and suggest if this solves your requirements.
